src/scaffold/ScaffoldView.tsx

```diff
@@ -118,119 +118,122 @@ const SecondPage: React.FC<PageProps> = ({ onPrevious }) => {
+                          handleOnClick={() => actions.updateOptions({ bundler: bundler.title })}
+                        />
+                      ))}
+                    </div>
+                  </div>
                 </div>
-              </div>
-            </div>
-            <div className="template-specific information" css={infosInputs}>
-              {templateOptions[0] !== 'spinner' &&
-              templateOptions[0] ? (
-                <React.Fragment>
-                  <p className="columnTitle">Provide Template Options</p>
-                  <div className="inputsWrapper">
-                    {state.templateOptions.map((option) => (
-                      <div className="extraItem capitalize" key={option[1].default}>
-                        <VSCodeTextField
-                          className="extraItemInput"
-                          placeholder={option[1].default}
-                          value={option[1].default}
-                          key={option[0]}
-                          onChange={(ev: any) =>
-                            actions.updateOptions({ dynamicOptionValues: [option[0], ev.target.value] })
-                          }>
-                          <p className="extraItemLabel">{option[0]}</p>
-                        </VSCodeTextField>
+                <div className="template-specific information" css={infosInputs}>
+                  {Object.keys(templateOptions).length > 0 ? (
+                    <React.Fragment>
+                      <p className="columnTitle">Provide Template Options</p>
+                      <div className="inputsWrapper">
+                        {Object.keys(templateOptions).map((option) => (
+                          <div className="extraItem capitalize" key={templateOptions[option].default}>
+                            <VSCodeTextField
+                              className="extraItemInput"
+                              placeholder={templateOptions[option].default}
+                              value={options.templateOptionsValues[option] || templateOptions[option].default}
+                              key={Object.keys(option)[0]}
+                              onChange={(ev: any) =>
+                                actions.updateOptions({
+                                  templateOptionsValues: { [option]: ev.target.value },
+                                })
+                              }>
+                              <p className="extraItemLabel">{option}</p>
+                            </VSCodeTextField>
+                          </div>
+                        ))}
                       </div>
-                    ))}
-                  </div>
-                </React.Fragment>
-              ) : templateOptions[0] === 'spinner' ? (
-                <div className="spinnerWrapper" css={spinner}>
-                  <div className="spinner" />
+                    </React.Fragment>
+                  ) : (
+                    ''
```

Comment:
What's this? Instead of `X ? Y : Z` you can also go for `X && Y`. Much simpler and exactly what you want.